Tests/TuistCacheTests/ContentHashing/CoreDataModelsContentHasherTests.swift

```diff
@@ -0,0 +1,91 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+
+final class CoreDataModelsContentHasherTests: TuistUnitTestCase {
+    private var sut: CoreDataModelsContentHasher!
```

Comment:
We usually call them `subject` but not a big deal

Reply:
I've change `sut` to `subject` in all tests